README.md

```diff
@@ -88,42 +88,89 @@ implementation group: 'io.harness', name: 'ff-java-server-sdk', version: '1.1.5.
 ```
 
 ### A Simple Example
+Here is a complete example that will connect to the feature flag service and report the flag value every 10 seconds until the connection is closed.  
+Any time a flag is toggled from the feature flag service you will receive the updated value.
 
-After installing the SDK, enter the SDK keys that you created for your environment. The SDK keys authorize your application to connect to the FF client. All features of the Java SDK are provided by the base class called `CfClient`.
+After installing the SDK, enter the SDK keys that you created for your environment. The SDK keys authorize your application to connect to the FF client. 
 
 ```java
-public class SimpleExample {
+package io.harness.ff.examples;
+
+import io.harness.cf.client.api.*;
+import io.harness.cf.client.connector.HarnessConnector;
+import io.harness.cf.client.dto.Target;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class GettingStarted {
+    // API Key - set this as an env variable
+    private static String apiKey = getEnvOrDefault("FF_API_KEY", "");
+
+    // Flag Identifier
+    private static String flagName = getEnvOrDefault("FF_FLAG_NAME", "harnessappdemodarkmode");
+
     public static void main(String[] args) {
+        log.info("Harness SDK Getting Started");
+
         try {
-            /**
-             * Put the API Key here from your environment
-             * Initialize the CfClient
-             */
-            String apiKey = System.getProperty("FF_API_KEY", "<default api key>");
-            String flagName = System.getProperty("FF_FLAG_NAME", "<default flag name>");
-            
-            CfClient cfClient = new CfClient(apiKey, Config.builder().build());
+            // Create a Feature Flag Client
+            CfClient cfClient = new CfClient(new HarnessConnector(apiKey));
```

Comment:
IMHO Under Simple Example it should only call new CFClient(SDKKEY), because this constructor will create under the hood HarnessConnector

Reply:
Make sense